src/backend/index.js

```diff
@@ -43,6 +43,15 @@ async function updateFeed(feed) {
   return currentFeed;
 }
 
+/**
+ * Invalidates a feed
+ * @param {Feed} feed
+ */
+async function invalidateFeed(feedData) {
+  const feed = Feed.parse(feedData);
+  await feed.setInvalid(feedData.reason || 'unknown reason');
```

Comment:
Can you file a follow-up issue so we can get the `reason` working properly?

Also, let's add a 

```js
logger.info(`Invalidating feed ${feedData.url} for the following reason: ${feedData.reason || unknown reason}`)
```

Reply:
#653